actionsystemtest/README.md

```diff
@@ -0,0 +1,108 @@
+# Action System Test
+
+Action System Test adds Capybara integration to your Rails application for
+acceptance testing. This allows you to test the entire user experience
+of your application rather than just your controllers, or just your models.
+
+Action System Test provides all of the setup out of the box for you to use
+Capybara with the Selenium Driver in your Rails application. Changing the
+default configuration is simple, yet flexible.
+
+## Examples
+
+### Usage
+
+By default Rails provides applications with system testing through Capybara
+and defaults to using the Selenium driver. The configuration set by Rails
+means that when you generate an application system tests will work out of
+the box, without you having to change any of the configuration requirements.
+
+Action System Test uses all the helpers from Capybara, but abstracts away the
+setup required to get running. Below is an example Action System Test.
+
+```ruby
+class UsersTest < ActionSystemTestCase
+  setup do
+    visit users_path
+  end
+
+  test 'creating a new user' do
+    click_on 'New User'
+
+    fill_in 'Name', with: 'Arya'
+
+    click_on 'Create User'
+
+    assert_text 'Arya'
+  end
+end
+```
+
+First we visit the +users_path+. From there we are going to use Action System
```

Comment:
We should be using _url's here, rather than _path's, since we're simulating full browser requests.